cmd/simulator/worker/worker.go

```diff
@@ -194,7 +194,7 @@ func (w *worker) sendTx(ctx context.Context, recipient common.Address, value *bi
 			GasFeeCap: feeCap,
 			GasTipCap: priorityFee,
 			Value:     value,
-			Data:      []byte{},
+			Data:      make([]byte, dynamicFeeTxDataN),
```

Comment:
just speculating: should we set random bytes here?

Reply:
Per @patrick-ogrady and for the sake of testing, zero bytes should be enough (testing the gas limit with simulated loads).

Reply:
Unless we do some processing with `Data` field in subnet-evm?